I'll need advice from @cms-sw/dqm-l2 on how to proceed. Even adding the subfolder and running the tests locally I get
```
Config file cmssw_cfg.py created
Task failed.
Traceback (most recent call last):
  File "/afs/cern.ch/user/t/tvami/public/PRsToCMSSW/l1Stage1DQMmodule/CMSSW_12_3_X_2022-02-22-1100/bin/slc7_amd64_gcc10/cmsswSequenceInfo.py", line 532, in <module>
    processseqs(seqset)
  File "/afs/cern.ch/user/t/tvami/public/PRsToCMSSW/l1Stage1DQMmodule/CMSSW_12_3_X_2022-02-22-1100/bin/slc7_amd64_gcc10/cmsswSequenceInfo.py", line 288, in processseqs
    for it in t.get(): # should only be one
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc10/external/python3/3.9.6-67e5cf5b4952101922f1d4c8474baa39/lib/python3.9/multiprocessing/pool.py", line 771, in get
    raise self._value
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc10/external/python3/3.9.6-67e5cf5b4952101922f1d4c8474baa39/lib/python3.9/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc10/external/python3/3.9.6-67e5cf5b4952101922f1d4c8474baa39/lib/python3.9/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "/afs/cern.ch/user/t/tvami/public/PRsToCMSSW/l1Stage1DQMmodule/CMSSW_12_3_X_2022-02-22-1100/bin/slc7_amd64_gcc10/cmsswSequenceInfo.py", line 272, in <lambda>
    tasks = [stp.map_async(lambda seq: (seq, inspectsequence(seq)), [seq]) for seq in seqs]
  File "/afs/cern.ch/user/t/tvami/public/PRsToCMSSW/l1Stage1DQMmodule/CMSSW_12_3_X_2022-02-22-1100/bin/slc7_amd64_gcc10/cmsswSequenceInfo.py", line 73, in inspectsequence
    raise Exception("cmsRun failed for cmsDriver command %s" % driverargs)
Exception: cmsRun failed for cmsDriver command ['cmsDriver.py', 'step3', '--conditions', 'auto:run2_data', '-s', 'DQM:@ExtraHLT', '--process', 'DUMMY', '--era', 'Run3', '--eventcontent', 'DQM', '--datatier', 'DQMIO', '--customise_commands', 'process.Tracer = cms.Service("Tracer")', '--filein', '/store/data/Run2018A/EGamma/RAW/v1/000/315/489/00000/004D960A-EA4C-E811-A908-FA163ED1F481.root', '-n', '0', '--python_filename', 'cmssw_cfg.py', '--no_exec']
[tvami@lxplus776 test]$ DQM:@miniAODDQM
We have determined that this is real data (if not, rerun cmsDriver.py with --mc)
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/nweek-02721/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_3_X_2022-02-21-1100/bin/slc7_amd64_gcc10/edmConfigDump", line 31, in <module>
    print(cmsProcess.dumpPython())
BrokenPipeError: [Errno 32] Broken pipe
entry /store/data/Run2018A/EGamma/RAW/v1/000/315/489/00000/004D960A-EA4C-E811-A908-FA163ED1F481.root
Step: DQM Spec: ['@miniAODDQM']
Config file cmssw_cfg.py created
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/nweek-02721/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_3_X_2022-02-21-1100/bin/slc7_amd64_gcc10/edmConfigDump", line 31, in <module>
    print(cmsProcess.dumpPython())
BrokenPipeError: [Errno 32] Broken pipe
```

--- a/DQM/SiStripMonitorClient/python/SiStripSourceConfigTier0_cff.py
+++ b/DQM/SiStripMonitorClient/python/SiStripSourceConfigTier0_cff.py
@@ -70,15 +70,24 @@ SiStripMonitorClusterBPTX = SiStripMonitorCluster.clone(
 
 from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
 stage2L1Trigger.toModify(SiStripMonitorClusterBPTX, 
-    BPTXfilter = dict(
-        stage2 = cms.bool(True),
-        l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
-        l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
-        ReadPrescalesFromFile = cms.bool(True)
-    ),
-    PixelDCSfilter = dict(stage2 = cms.bool(True)),
-    StripDCSfilter = dict(stage2 = cms.bool(True))
-)
+                         BPTXfilter = dict(
+                             stage2 = cms.bool(True),
+                             l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                             l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                             ReadPrescalesFromFile = cms.bool(True)
+                         ),
+                         PixelDCSfilter = dict(
+                             stage2 = cms.bool(True),
+                             l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                             l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                             ReadPrescalesFromFile = cms.bool(True)),
+                         StripDCSfilter = dict(
+                             stage2 = cms.bool(True),
+                             l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                             l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                             ReadPrescalesFromFile = cms.bool(True)
+                         )
+                        )
 
 # refitter ### (FIXME rename, move)
 from DQM.SiPixelMonitorTrack.RefitterForPixelDQM import *

--- a/DQMOffline/Trigger/plugins/BPHMonitor.cc
+++ b/DQMOffline/Trigger/plugins/BPHMonitor.cc
@@ -1262,6 +1262,10 @@ void BPHMonitor::fillDescriptions(edm::ConfigurationDescriptions& descriptions) 
   genericTriggerEventPSet.add<bool>("l1BeforeMask", true);
   genericTriggerEventPSet.add<unsigned int>("verbosityLevel", 0);
   genericTriggerEventPSet.add<bool>("stage2", false);
+  genericTriggerEventPSet.add<edm::InputTag>("l1tAlgBlkInputTag", edm::InputTag("gtStage2Digis"));
+  genericTriggerEventPSet.add<edm::InputTag>("l1tExtBlkInputTag", edm::InputTag("gtStage2Digis"));
+  genericTriggerEventPSet.add<bool>("ReadPrescalesFromFile", false);
+
   desc.add<edm::ParameterSetDescription>("numGenericTriggerEventPSet", genericTriggerEventPSet);
   desc.add<edm::ParameterSetDescription>("denGenericTriggerEventPSet", genericTriggerEventPSet);
 

--- a/DQMOffline/Trigger/plugins/JetMonitor.cc
+++ b/DQMOffline/Trigger/plugins/JetMonitor.cc
@@ -669,6 +669,9 @@ void JetMonitor::fillDescriptions(edm::ConfigurationDescriptions& descriptions) 
   genericTriggerEventPSet.add<bool>("errorReplyHlt", false);
   genericTriggerEventPSet.add<unsigned int>("verbosityLevel", 1);
   genericTriggerEventPSet.add<bool>("stage2", false);
+  genericTriggerEventPSet.add<edm::InputTag>("l1tAlgBlkInputTag", edm::InputTag("gtStage2Digis"));
+  genericTriggerEventPSet.add<edm::InputTag>("l1tExtBlkInputTag", edm::InputTag("gtStage2Digis"));
+  genericTriggerEventPSet.add<bool>("ReadPrescalesFromFile", false);
 
   desc.add<edm::ParameterSetDescription>("numGenericTriggerEventPSet", genericTriggerEventPSet);
   desc.add<edm::ParameterSetDescription>("denGenericTriggerEventPSet", genericTriggerEventPSet);

--- a/DQMOffline/Trigger/plugins/METMonitor.cc
+++ b/DQMOffline/Trigger/plugins/METMonitor.cc
@@ -392,6 +392,9 @@ void METMonitor::fillDescriptions(edm::ConfigurationDescriptions& descriptions) 
   genericTriggerEventPSet.add<bool>("errorReplyHlt", false);
   genericTriggerEventPSet.add<unsigned int>("verbosityLevel", 1);
   genericTriggerEventPSet.add<bool>("stage2", false);
+  genericTriggerEventPSet.add<edm::InputTag>("l1tAlgBlkInputTag", edm::InputTag("gtStage2Digis"));
+  genericTriggerEventPSet.add<edm::InputTag>("l1tExtBlkInputTag", edm::InputTag("gtStage2Digis"));
+  genericTriggerEventPSet.add<bool>("ReadPrescalesFromFile", false);
 
   desc.add<edm::ParameterSetDescription>("numGenericTriggerEventPSet", genericTriggerEventPSet);
   desc.add<edm::ParameterSetDescription>("denGenericTriggerEventPSet", genericTriggerEventPSet);

--- a/DQMOffline/Trigger/plugins/MuonMonitor.cc
+++ b/DQMOffline/Trigger/plugins/MuonMonitor.cc
@@ -371,6 +371,9 @@ void MuonMonitor::fillDescriptions(edm::ConfigurationDescriptions& descriptions)
   genericTriggerEventPSet.add<bool>("andOrL1", false);
   genericTriggerEventPSet.add<bool>("l1BeforeMask", false);
   genericTriggerEventPSet.add<bool>("stage2", false);
+  genericTriggerEventPSet.add<edm::InputTag>("l1tAlgBlkInputTag", edm::InputTag("gtStage2Digis"));
+  genericTriggerEventPSet.add<edm::InputTag>("l1tExtBlkInputTag", edm::InputTag("gtStage2Digis"));
+  genericTriggerEventPSet.add<bool>("ReadPrescalesFromFile", false);
 
   genericTriggerEventPSet.add<std::vector<std::string>>("l1Algorithms", {});
 

--- a/DQMOffline/Trigger/python/BPHMonitor_cfi.py
+++ b/DQMOffline/Trigger/python/BPHMonitor_cfi.py
@@ -118,6 +118,12 @@ hltBPHmonitoring.denGenericTriggerEventPSet.verbosityLevel = cms.uint32(0)
 from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
 stage2L1Trigger.toModify(hltBPHmonitoring,
                          stageL1Trigger = 2,
-                         numGenericTriggerEventPSet = dict(stage2 = cms.bool(True)),
-                         denGenericTriggerEventPSet = dict(stage2 = cms.bool(True)))
+                         numGenericTriggerEventPSet = dict(stage2 = cms.bool(True),
+                                                           l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           ReadPrescalesFromFile = cms.bool(True)),
+                         denGenericTriggerEventPSet = dict(stage2 = cms.bool(True),
+                                                           l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           ReadPrescalesFromFile = cms.bool(True)))
 

--- a/DQMOffline/Trigger/python/JetMonitor_cfi.py
+++ b/DQMOffline/Trigger/python/JetMonitor_cfi.py
@@ -41,6 +41,12 @@ hltJetMETmonitoring.denGenericTriggerEventPSet.verbosityLevel = cms.uint32(1)
 
 from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
 stage2L1Trigger.toModify(hltJetMETmonitoring,
-                         numGenericTriggerEventPSet = dict(stage2 = cms.bool(True)),
-                         denGenericTriggerEventPSet = dict(stage2 = cms.bool(True)))
+                         numGenericTriggerEventPSet = dict(stage2 = cms.bool(True),
+                                                           l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           ReadPrescalesFromFile = cms.bool(True)),
+                         denGenericTriggerEventPSet = dict(stage2 = cms.bool(True),
+                                                           l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           ReadPrescalesFromFile = cms.bool(True)))
 

--- a/DQMOffline/Trigger/python/METMonitor_cfi.py
+++ b/DQMOffline/Trigger/python/METMonitor_cfi.py
@@ -40,6 +40,12 @@ hltMETmonitoring.denGenericTriggerEventPSet.hltPaths      = cms.vstring();
 
 from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
 stage2L1Trigger.toModify(hltMETmonitoring,
-                         numGenericTriggerEventPSet = dict(stage2 = cms.bool(True)),
-                         denGenericTriggerEventPSet = dict(stage2 = cms.bool(True)))
+                         numGenericTriggerEventPSet = dict(stage2 = cms.bool(True),
+                                                           l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           ReadPrescalesFromFile = cms.bool(True)),
+                         denGenericTriggerEventPSet = dict(stage2 = cms.bool(True),
+                                                           l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           ReadPrescalesFromFile = cms.bool(True)))
 

--- a/DQMOffline/Trigger/python/MuonMonitor_cfi.py
+++ b/DQMOffline/Trigger/python/MuonMonitor_cfi.py
@@ -41,6 +41,12 @@ hltMuonmonitoring.denGenericTriggerEventPSet.verbosityLevel = cms.uint32(1)
 
 from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
 stage2L1Trigger.toModify(hltMuonmonitoring,
-                         numGenericTriggerEventPSet = dict(stage2 = cms.bool(True)),
-                         denGenericTriggerEventPSet = dict(stage2 = cms.bool(True)))
+                         numGenericTriggerEventPSet = dict(stage2 = cms.bool(True),
+                                                           l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           ReadPrescalesFromFile = cms.bool(True)),
+                         denGenericTriggerEventPSet = dict(stage2 = cms.bool(True),
+                                                           l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           l1tExtBlkInputTag = cms.InputTag("gtStage2Digis"),
+                                                           ReadPrescalesFromFile = cms.bool(True)))
 

